MekHQ/src/mekhq/campaign/stratcon/StratconContractInitializer.java

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (c) 2019 The Megamek Team. All rights reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MekHQ.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.campaign.stratcon;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import megamek.common.Compute;
+import mekhq.MekHQ;
+import mekhq.campaign.Campaign;
+import mekhq.campaign.force.Force;
+import mekhq.campaign.mission.AtBContract;
+import mekhq.campaign.mission.AtBDynamicScenario;
+import mekhq.campaign.mission.Mission;
+import mekhq.campaign.mission.Scenario;
+import mekhq.campaign.mission.ScenarioTemplate;
+import mekhq.campaign.mission.ScenarioForceTemplate.ForceAlignment;
+import mekhq.campaign.mission.atb.AtBScenarioModifier;
+import mekhq.campaign.stratcon.StratconContractDefinition.ObjectiveParameters;
+
+/**
+ * This class handles StratCon state initialization when a contract is signed.
+ */
+public class StratconContractInitializer {
+    public static final int NUM_LANCES_PER_TRACK = 3;
+
+    /**
+     * Initializes the campaign state given a contract, campaign and contract definition
+     */
+    public static void initializeCampaignState(AtBContract contract, Campaign campaign, StratconContractDefinition contractDefinition) {
+        StratconCampaignState campaignState = new StratconCampaignState(contract);
+        campaignState.setBriefingText(contractDefinition.getBriefing() + "<br/>" + generateCommandLevelText(contract));
```

Comment:
Not certain, but `<br>` should work just fine

Reply:
Probably, but it's a personal preference.